### PR TITLE
Changed the order of the submit button to match capa action bar on pl…

### PR DESCRIPTION
**Pull request by staubina**
…atform.

### Description

Change the order of the tab focus for the action bar on drag and drop problems

### Sandbox
https://staubina-dnd2.sandbox.edx.org/courses/course-v1:edX+DemoX+Demo_Course/courseware/3189bd686c9a4112b8e3bdf1a437273c/54bd378088f844238b29560a8a79fd61/1?activate_block_id=block-v1%3AedX%2BDemoX%2BDemo_Course%2Btype%40vertical%2Bblock%40d56c97d8c5074b30a0b2d181994f01b4
- [x] Build a sandbox for your branch and add a link here

### Testing
- [x] i18n
- [x] RTL
- [x] Unit, integration, acceptance tests as appropriate

### Reviewers
If you've been tagged for review, please check your corresponding box once you've given the :+1:.
- [x] Code review: @cahrens
- [x] Code review: @jlajoie
- [ ] Accessibility review: @cptvitamin

FYI: @sstack, @sanfordstudent  

### Post-review
- [x] Rebase and squash commits

### CODE REVIEW

**Comment by staubina** [comment]
@jlajoie, @cahrens, and @cptvitamin Can you please review this PR when you have time.  I am looking into the test failures.

File: .travis.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ install:
     - "pip install selenium==2.53.0"
     - "pip uninstall -y xblock-drag-and-drop-v2"
     - "python setup.py sdist"
-    - "pip install dist/xblock-drag-and-drop-v2-2.0.15.tar.gz"
+    - "pip install dist/xblock-drag-and-drop-v2-2.0.16.tar.gz"
```
Comment (staubina) [inline .travis.yml]: Note: this was added to be able to build on to the sandbox and might be removed into another PR when we are ready to release

File: setup.py [hunk 2]
```diff
@@ -23,7 +23,7 @@ def package_data(pkg, root_list):
 
 setup(
     name='xblock-drag-and-drop-v2',
-    version='2.0.15',
+    version='2.0.16',
```
Comment (staubina) [inline setup.py]: Note: this was added to be able to build on to the sandbox and might be removed into another PR when we are ready to release

File: .travis.yml (hunk 1 above)
Comment (cahrens) [inline .travis.yml]: It seems like these should be in sync though. Why would it be removed?

Comment (staubina) [inline .travis.yml]: As we discussed they should absolutely be in sync.  I would remove it so that @sanfordstudent and I would only need to release one version.

**Comment by cahrens** [comment]
👍

**Comment by staubina** [comment]
@jlajoie and @cptvitamin, please review this PR today if you can.

**Comment by jlajoie** [comment]
:+1:

**Comment by cahrens** [comment]
I don't think we need @cptvitamin to review this. It's a simple change to the tab order to be consistent with capa.

**Comment by cptvitamin** [comment]
👍   thanks for taking care of this.
